### PR TITLE
first page

### PR DESCRIPTION
**Pull request by SoheilNK**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,11 +44,38 @@
...
+            <span>Q</span>
+            <span></span>
+            <span>T</span>
+            <span>e</span>
+            <span>s</span>
+            <span>t</span>
+        </h1>
+        
+
+        <div>This test has to sections</div>
```
Comment (leonchisholm) [inline index.html]: "to" should be "two"

File: index.html [hunk 2]
```diff
@@ -40,11 +44,38 @@
...
+        <div id="sections">
+              <div id="sec1">In the first section you will encounter mathematical questions and you need to answer them in a certain amount of time.</div>  
+             <div id="sec2">For the second section, app will show you some pictures and you need to use the provided shapes and drag and drop them on the specific area to recreate the pattern.</div>
```
Comment (leonchisholm) [inline index.html]: "For the second section, the app..."

And similarly to my comment above - shorter sentences are easier to understand. "For the second section, the app will show you some patterns. Recreate each pattern by dragging and dropping the provided shapes onto the given grid."

File: index.html [hunk 3]
```diff
@@ -40,11 +44,38 @@
...
+            <span>T</span>
+            <span>e</span>
+            <span>s</span>
+            <span>t</span>
+        </h1>
+        
+
+        <div>This test has to sections</div>
+        <div id="sections">
+              <div id="sec1">In the first section you will encounter mathematical questions and you need to answer them in a certain amount of time.</div>
```
Comment (leonchisholm) [inline index.html]: This explanation is fine, but from a readability stand-point maybe use shorter sentences? Like: "In the first section you will encounter mathematical questions. Answer them in the allotted time." (Or "Please answer them..." if you prefer)

File: index.css [hunk 4]
```diff
@@ -78,3 +105,78 @@ label {
...
+                0 3px 0 #CCC,
+                0 4px 0 #CCC,
+                0 5px 0 #CCC,
+                0 6px 0 #CCC,
+                0 7px 0 #CCC,
+                0 8px 0 #CCC,
+                0 9px 0 #CCC,
+                0 50px 25px rgba(0, 0, 0, 0.2);
+        }
+    }
```
Comment (leonchisholm) [inline index.css:194]: Nice to see your work in css! Just out of curiosity, where did you learn how to do this?

File: index.html [hunk 5]
```diff
@@ -40,11 +44,38 @@
...
+            <span>e</span>
+            <span>l</span>
+            <span>c</span>
+            <span>o</span>
+            <span>m</span>
+            <span>e</span>
+            <span> </span>
+            <span>t</span>
+            <span>o</span>
+            <span></span>
```
Comment (leonchisholm) [inline index.html]: "Welcome to the IQ Test" - I'd include a "the" here.

**Review by leonchisholm — CHANGES_REQUESTED** [review]
Great work, Soheil! Left some comments. :-)

File: index.css (hunk 4 above)
Comment (SoheilNK) [reply index.css:194]: Thanks, Leon. I learn it from our good friend 'Google'  :D 
I just searched and found these,
https://www.geeksforgeeks.org/how-to-create-jumping-text-3d-animation-effect-using-css/
https://www.geeksforgeeks.org/how-to-create-fade-in-effect-on-page-load-using-css/

**Comment by SoheilNK** [comment]
Thank you, Leon. 
I applied the changes. 
Could you please have a look at the 'about' page too?

**Pull request merged**